sssom/util.py

```diff
@@ -521,10 +521,15 @@ def merge_msdf(
     msdf_with_meta = [inject_metadata_into_df(msdf) for msdf in msdfs]
 
     # merge df [# 'outer' join in pandas == FULL JOIN in SQL]
+    # df_merged = reduce(
+    #     lambda left, right: left.merge(right, how="outer", on=list(left.columns)),
+    #     [msdf.df for msdf in msdf_with_meta if msdf.df is not None],
+    # )
+    # Concat is an alternative to merge when columns are not the same.
     df_merged = reduce(
-        lambda left, right: left.merge(right, how="outer", on=list(left.columns)),
+        lambda left, right: pd.concat([left, right], axis=0, ignore_index=True),
```

Comment:
I trust you here, dont understand the implications :)

Reply:
I've tested it with our unit test file, did a `df1.equals(df2)` which equates to `True`